src/onnx/parse_convolution.cpp

```diff
@@ -149,9 +193,58 @@ struct parse_convolution : op_parser<parse_convolution>
 
         recalc_conv_attributes(values, kdims);
 
+        instruction_ref ret;
+        // parse a_zero_point and b_zero_point values
+        auto is_quant_conv = opd.op_name == "quant_convolution";
+        auto is_input_bias  = args.size() > 2;
+        auto is_weight_bias = args.size() > 3;
+
+        auto x_zp = get_zero_point(x, 2, is_quant_conv, args);
+        auto w_zp = get_zero_point(weights, 3, is_quant_conv, args);
+
         op.from_value(values);
-        auto l1 = info.add_instruction(op, l0, args[1]);
-        return info.add_bias(args, l1, 1);
+
+        ret = info.add_instruction(op, x, weights);
+
+        // Handle quant_conv residuals between input/weights to avoid overflow
+        if(is_quant_conv)
+        {
+            if(not is_symmetric_zero_point(x_zp) and is_input_bias)
+            {
+                auto out_zp_1 = info.add_common_op("quant_convolution", x_zp, weights);
```

Comment:
these additional quant_conv ops should use the same attributes (ie. padding, etc.) as the original quant_conv

Reply:
TODO: Share same attributes between quant_convolutions for asymmetric  inputs

Reply:
Done